howto/install/deploy-bare-metal.md

```diff
@@ -0,0 +1,93 @@
+To deploy Anbox Cloud on a public cloud (such as AWS, Azure or Google) or using MAAS or OpenStack, see the instructions in [Deploy Anbox Cloud with Juju](https://discourse.ubuntu.com/t/deploy-anbox-cloud-with-juju/17744).
+
+Alternatively, you can follow the instructions in this document to use the [manual cloud provider](https://jaas.ai/docs/manual-cloud) that Juju offers. This method allows you to deploy Anbox Cloud with Juju on a set of SSH connected machines.
+
+## Prerequisites
+
+Before you start the installation, ensure that you have the required prerequisites:
+
+* At least three Ubuntu 18.04 LTS or 20.04 LTS machines. See [Minimum hardware](https://discourse.ubuntu.com/t/requirements/17734#minimum-hardware) for details and recommendations.
```

Comment:
Is "at least three" correct?

Reply:
Technically you can run Anbox cloud on a single machine (that's what the appliance does), however in production, the more machines you can use, the better
Maybe we should reflect that in the docs ?

Reply:
Lets not mention 18.04 anymore. We need to get customers at minimum on 20.04 even if we still support 18.04 because we have existing customers on it.